packages/okta-angular/src/okta/models/okta.config.ts

```diff
@@ -21,6 +21,7 @@ export interface OktaConfig {
   scope?: string;
   responseType?: string;
   onAuthRequired?: AuthRequiredFunction;
+  testing?: Object;
```

Comment:
While correct, I think we should be a bit more verbose here.

Lets create a new interface, `TestingObject` that has known key/value pairs we'd expect:

```ts
interface TestingObject {
  disableHttpsCheck: boolean;
}

// Then set the testing optional to:
testing?: TestingObject
```